test_search.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import os
+
+import requests
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Credentials from environment variables
+CONFLUENCE_URL = os.getenv("CONFLUENCE_URL")
+CONFLUENCE_USERNAME = os.getenv("CONFLUENCE_USERNAME")
+CONFLUENCE_API_TOKEN = os.getenv("CONFLUENCE_API_TOKEN")
+SPACE_KEY = os.getenv("CONFLUENCE_SPACE_KEY")
+
+# Check if environment variables are loaded
+if not all([CONFLUENCE_URL, CONFLUENCE_USERNAME, CONFLUENCE_API_TOKEN, SPACE_KEY]):
+    print("Error: Missing environment variables. Please create a .env file with the required variables.")
+    exit(1)
+
+# Test with just a query
+print("Test 1: Search with simple query")
+query = "test"
+response = requests.get(
+    f"{CONFLUENCE_URL}/wiki/api/v2/search",
```

Comment:
I can not find this endpoint in the docs (https://developer.atlassian.com/cloud/confluence/rest/v2/intro/#about) and it also does not work for my instance.

Reply:
Let me check it

Reply:
Indeed